example/docker-compose/etc/tempo-local.yaml

```diff
@@ -38,7 +38,7 @@ storage:
       bloom_filter_false_positive: .05 # bloom filter false positive rate.  lower values create larger filters but fewer false positives
       index_downsample: 10             # number of traces per index record
     local:
-      path: /tmp/tempo
+      path: /tmp/tempo/blocks
```

Comment:
any particular reason to change this?

Reply:
Oh yes this would need to change because the WAL is also populated in this folder, and we're picking it up as a tenant 😄 

```
level=error ts=2020-11-20T11:53:06.9741943Z caller=tempodb.go:399 msg="error polling blocklist" tenantID=traces err="invalid UUID length: 13"
level=error ts=2020-11-20T11:53:06.9787077Z caller=tempodb.go:399 msg="error polling blocklist" tenantID=wal err="invalid UUID length: 9"
```